braidpool_spec.md

```diff
@@ -180,13 +287,33 @@ median of the descendant cohort's timestamps. Here we use only the timestamps
 as witnessed by descendants, not the claimed broadcast time by the miner in
 `Uncommitted Metadata`.
 
+[kp]
+
+There is new terminology here which the reader has not been introduced
+to. Things like cohort and cohort time. We should explain these terms before
+using them.
+
+[/kp]
+
 (FIXME: Is it appropriate to apply this factor to even blockchain-like beads?)
 
 As $T_C$ grows, the value of shares decreases. Therefore an attacker attempting
 to reorganize transactions or execute a selfish mining attack will see the value
 of his shares decrease in an appropriate way corresponding to how likely it is
 that he generates an orphan and reduces the profit of the pool.
 
+[kp]
+
+Reducing share weight is problematic. 
+
+A miner, does work X over time, the entire work done by braidpool is Y by the
+time BP finds a bitoin block. The incentives compatible thing to do is to give
+the miner X/Y of the reward. Otherwise, we incentive miners to leave BP as soon
+as a block is found and come back after a few days, when their work will receive
+more reward.
+
+[/kp]
+
```

Comment:
...give the miner $X/(X+Y)$ of the reward (this is exactly the proportional payout mechanism).